### ISSUE TITLE
Grab the old features pages from the WayBackMachine --> Update for GeoCoreCE

### ISSUE THREAD

**Issue by vicos59** [geodesicsolutions-community/geocore-community#86]
Grab the old features pages from the WayBackMachine, clean them up and post them on the .org site. Beats rewriting all that stuff from scratch.

https://web.archive.org/web/20200525190920/https://geodesicsolutions.com/add-ons/free-add-ons.html

https://web.archive.org/web/20200512051130/https://geodesicsolutions.com/geocore-software/geocore-admin.html

https://web.archive.org/web/20200512045706/https://geodesicsolutions.com/geocore-software/geocore-themes.html

https://web.archive.org/web/20200523100023/https://geodesicsolutions.com/classified-ad-software-and-auction-features/features-list.html

**jonyo adds:**

 Sure, it wouldn't hurt, though we'd probably need to re-make the screenshots so it has the updated community edition name and all that.

If you do that, be sure to record if you use any specific tools (like any software that automatically makes mobile/desktop/tablet views or something) so we can easily re-create them later if needed.

For the icons, something I have seen recently is [box icons](https://boxicons.com/) - entirely free icon set. It may be a good resource for the website when we need icons, and in the software if someone wants to update the look.

That is a good resource as they provide SVG's which allow easily changing the color with CSS.

**Comment by vicos59** [comment]
Maybe we should consider just adding the "features" list to the Github Wiki. Would be easier to maintain, and if we might one day migrate the wiki here from the .org site... Something to think about.

In any case need to grab the original code soon in case it disappears one day.

**Comment by jonyo** [comment]
@vicos59 that is one option, but does have its downsides:

You can't do much with layout in the wiki.

As soon as you get into the GitHub site I wonder if it would be info overload for someone new to it.  Especially since this is the feature list, it is probably one of the first things someone will look at.

I agree it would make it easier to maintain though.

**Comment by vicos59** [comment]
This seems to be "The Guy"

https://github.com/hartator/wayback-machine-downloader

**Comment by vicos59** [comment]
@jonyo Do you roughly recall the month/year you were last called in to fix up the web site after they moved to a new server? I am trying set my time range to grab the right stuff. The WayBackMachine grabs different elements on various date. So, you might have to scrape an entire year's worth of archives to get all the files you need.

**Comment by vicos59** [comment]
Grabbed the pages using the script. Still a few support files missing. These files were originally generated by Joomla.

Removed the header, footer and menu. Restructuring the folder layout so everything is contained in one base folder.

Needs more cleanup and figure out how to integrate it into main site.

**Comment by vicos59** [comment]
@jonyo Are the 'Language Packs' going to be available? If not, I'll remove from the features page. It says to download them in the Client Area.

https://web.archive.org/web/20200512045156/https://geodesicsolutions.com/add-ons/free-add-ons/386-language-packs.html

Matters not to me if they are or are not.

**Comment by vicos59** [comment]
@jonyo Just issued a pull request https://github.com/geodesicsolutions-community/website/pull/13 for what I have finished so far. Became more involved than expected. Will get back to this when I can.

**Comment by jonyo** [comment]
> @jonyo Are the 'Language Packs' going to be available? If not, I'll remove from the features page. It says to download them in the Client Area.
> 
> https://web.archive.org/web/20200512045156/https://geodesicsolutions.com/add-ons/free-add-ons/386-language-packs.html
> 
> Matters not to me if they are or are not.

No, don't have the old language packs, they were not something they kept in the git source code.

**Comment by vicos59** [comment]
I had forgotten how tedious it is to edit numerous HTML files vs using a CMS or script-generated pages.

I've been thinking about this and the entire website is high maintenance right now. As far as the software features pages defined in this Issue, I now think it would just be best put them right into the Wiki where they can be easily maintained. Frankly, I am thinking it might be best to just move the entire wiki to the top level and let it be the website. Dokuwiki seems to be active and maintained. 

It would take me another 2-3 hours to finish cleaning up these HTML files. In the same time or less, I could probab'y just copy and paste it into the Wiki.

Just something to think about. I am putting this Issue on hold for now until we decide the best way forward.

**Comment by jonyo** [comment]
> I now think it would just be best put them right into the Wiki where they can be easily maintained

The down side like I've mentioned before, is that it makes it more difficult to style it.  Of course you've discovered the up sides, being easier to maintain and such.

> Frankly, I am thinking it might be best to just move the entire wiki to the top level and let it be the website. Dokuwiki seems to be active and maintained.

For simple websites where the main part does not change often, I'd prefer static HTML files over a CMS, the CMS ends up being overkill.  Since once it is made you don't touch it much.

But a feature list we probably would.  So now I'm going to contradict myself 😆 - maybe for the feature list we use something.

One option is we could look into github pages, I've used that years ago.  It might be a good option.

**Comment by jonyo** [comment]
Hmm, so I kind of convinced myself, maybe github pages is the way to go.

But wait, what about the wiki content...  And the dynamic parts like dynamic upgrade instructions...  When I have time I'll look into it more.

**Comment by vicos59** [comment]
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site

> You can customize the domain name of your GitHub Pages site.

If only we had a good domain name we could use for this....🤔
